collections/events.js

```diff
@@ -182,7 +182,7 @@ Events.updateGroups = function(eventId) {
 
 
 Meteor.methods({
-	saveEvent: function(eventId, changes, updateReplicas, sendNotifications) {
+	saveEvent: function(eventId, changes, updateReplicas, sendNotifications, addNotificationMessage) {
```

Comment:
At this point, we need to at least `check(addNotificationMessage, String)`. Also trim and truncate to a sane length like 1k bytes.

What's the reason for calling the variable `addNotificationMessage` instead of just `message`? Do we expect there to be other messages? It didn't bother me on the client side, but here it does :-)